lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -3460,25 +3460,39 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
 
     describe "TRANS_NEW_CREATE_MINT_SCRIPTS - I can mint and burn with different policy scripts" $ do
         let scenarios =
-                  [ ( "all", [json|{ "all": [ "cosigner#0" ] }|] )
-                  , ( "any", [json|{ "any": [ "cosigner#0" ] }|] )
-                  , ( "some", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0" ]} }|] )
-                  , ( "all, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_until": 57297561 } ] }|] )
-                  , ( "any, active_until 57297561", [json|{ "any": [ "cosigner#0",  { "active_until": 57297561 } ] }|] )
-                  , ( "some, active_until 57297561", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_until": 57297561 } ]} }|] )
-                  , ( "all, active_from 10", [json|{ "all": [ "cosigner#0",  { "active_from": 10 } ] }|] )
-                  , ( "any, active_from 10", [json|{ "any": [ "cosigner#0",  { "active_from": 10 } ] }|] )
-                  , ( "some, active_from 10", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_from": 10 } ]} }|] )
-                  , ( "all, active_from 10, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_from": 10 }, { "active_until": 57297561 } ] }|] )
-                  , ( "any, active_from 10, active_until 57297561", [json|{ "any": [ "cosigner#0",  { "active_from": 10 }, { "active_until": 57297561 } ] }|] )
```

Comment:
this was illegal script `ReduntantTimelocks` https://github.com/input-output-hk/cardano-addresses/blob/master/core/lib/Cardano/Address/Script.hs#L405